### DISCUSSION TITLE
Where's the jar file? Do I have to compile it?

### DISCUSSION THREAD

**Discussion by steamy-steve** [headlesshq/headlessmc#114] [Q&A]
Hi,
the description mentions the headlessmc jar file, but I can't find it anywhere.

Usually, when I follow a link to github for an add-on or similar, there's always the final file to download. But here I can't find it.
Do I have to compile it on my own?

**Comment by 3arthqu4ke** [comment]
https://github.com/3arthqu4ke/headlessmc/releases
